pyproject.toml

```diff
@@ -10,11 +10,9 @@ build-backend = "flit_core.buildapi"
 module = "poliastro"
 author = "Juan Luis Cano Rodríguez"
 author-email = "hello@juanlu.space"
-# TODO: No short description?
 description-file = "README.rst"
 home-page = "https://github.com/poliastro/poliastro/"
-# TODO: Keywords cannot contain spaces?
-keywords = "aero aerospace engineering astrodynamics orbits kepler orbital mechanics"
+keywords = "aero, aerospace, engineering, astrodynamics, orbits, kepler, orbital mechanics"
```

Comment:
It looks like distutils & setuptools do a simple `.split(',')`, so I suspect you want no spaces after the commas. I'll try to sand the rough edges off this soon.

Reply:
Thanks @takluyver! That should be good enough, will change it now.